ui/src/ScholarReader.tsx

```diff
@@ -182,6 +196,20 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
     }
   }
 
+  async addToLibrary(paperId: string, paperTitle: string) {
+    if (this.props.paperId !== undefined) {
+      await api.addLibraryEntry(
```

Comment:
we should do a try/catch to show an error message if for some reason this call fails